### DISCUSSION TITLE
Sorry for the absence!

### DISCUSSION THREAD

**Discussion by AprilSylph** [AprilSylph/XKit-Rewritten#1403] [Tech Office]
Apologies that I've become a little bit of an absent mother to this project. My energy levels lately have not been what I like them to be. But, I think I might have myself figured out this time! I just need to be better to myself, and myself will be better to me in turn. (Yeah, I'm gonna keep that sentence. It's silly, but I think it's accurate.)

I would _like_ to review at least one PR per calendar day, and do housekeeping on my various projects when my review queue is empty. I managed to get in some reviews tonight, and I genuinely hope I can make it a habit again.

It's been over two months since the last addon release by now. What do you think the release cadence _should_ be? Every week, every month? At the very, very least, I shouldn't be allowing merged changes to sit unreleased for this long.

If you have any other thoughts related to my recent lack of involvement, those are welcome here too.

Let me know what you think. :purple_heart:

**Comment by marcustyphoon** [comment]
Oh, no worries! Glad you're feeling well at the moment ❤️ 

I don't think I've ever had a particular release cadence in mind; as we sort of mentioned previously, I think it depends on the content! If you had merged #1351, for example, I figured I would have pinged you at some point suggesting a release. But when it's background code changes and bugfixes for things that I doubt anyone but me would have even noticed, I mean, eh! Honestly if this were XKit 7, where pushing any update causes a visible notification to the user, I would be worried (casually; who cares, but) about the opposite problem wherein people might go looking for the functionality change when there isn't really one!
